### PR TITLE
Create database and tables

### PR DESCRIPTION
**Pull request by ghifo**

### CODE REVIEW

File: Week1/databases/index.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+connection.query('CREATE DATABASE IF NOT EXISTS world', handleErrorAndData('Query OK'));
+
+connection.query('use world', handleErrorAndData('Database changed'));
+
+connection.query(
+  ` CREATE TABLE IF NOT EXISTS country (
+    Code CHAR(3) NOT NULL DEFAULT '',
+    Name CHAR(52) NOT NULL DEFAULT '',
+    Continent enum('Asia','Europe','North America','Africa','Oceania','Antarctica','South America') NOT NULL DEFAULT 'Asia',
```
Comment (gajduk) [inline Week1/databases/index.js]: using enums 👍

File: Week1/databases/index.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+connection.query(
+  ` CREATE TABLE IF NOT EXISTS city (
+    ID INT(11) NOT NULL AUTO_INCREMENT,
+    Name CHAR(35) NOT NULL DEFAULT '',
+    Country_Code CHAR(3) NOT NULL DEFAULT '',
+    District CHAR(20) NOT NULL DEFAULT '',
+    Population INT(11) NOT NULL DEFAULT '0',
+    PRIMARY KEY (ID),
+    FOREIGN KEY (Country_Code) REFERENCES country (Code)
```
Comment (gajduk) [inline Week1/databases/index.js]: Foreign Key 👍

File: Week1/databases/newversion.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  }
+};
+
+connection.query('CREATE DATABASE IF NOT EXISTS world', cb());
+
+connection.query('use world', cb());
+
+connection.query(
+  ` CREATE TABLE IF NOT EXISTS country (
+    Code CHAR(3) NOT NULL DEFAULT '',
```
Comment (gajduk) [inline Week1/databases/newversion.js]: like that the definition is detailed and split into different lines, makes it much nicer to read 👍

Comment (ghifo) [reply Week1/databases/newversion.js]: I think I love DATABASES :)

**Pull request closed without merging**